src/commands/suspend.ts

```diff
@@ -76,7 +76,7 @@ export const suspendCommand: CommandDef = {
         .addFields(
           {
             name: 'What happened?',
-            value: `${mod} has suspended ${user}.`
+            value: `${mod} has suspended **${user.user.username}** (${user}).`
```

Comment:
The fact its `user.user.username` is **weird**, our `user` might be mis-named, but we can handle this later.

Reply:
Yeh - this was an issue I ran into on my bot. 

The `user` variable is actually the `member` object, so the variable *should* be renamed as such.